pkg/systemd/terminal.jsx

```diff
@@ -72,27 +87,27 @@ const _ = cockpit.gettext;
 
         setCookie(key, value) {
             const cookie = key + "=" + encodeURIComponent(value) +
-                         "; path=/; expires=Sun, 16 Jul 3567 06:23:41 GMT";
+                         "; path=/; Max-Age=0;";
             document.cookie = cookie;
         }
 
         onPlus() {
             this.setState((state, _) => {
-                this.setCookie("size_cookie", state.size + 1);
+                cockpit.localStorage.setItem('terminal-font-size', state.size + 1);
                 return { size: state.size + 1 };
             });
         }
 
         onMinus() {
             this.setState((state, _) => {
-                this.setCookie("size_cookie", state.size - 1);
+                cockpit.localStorage.setItem('terminal-font-size', state.size - 1);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18499-20230313-194244-ef71b886-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)